docs/tutorials/10-Hyperparameter Tuning.ipynb

```diff
@@ -1379,7 +1379,7 @@
     ")\n",
     "with warnings.catch_warnings():\n",
     "    warnings.simplefilter(\"ignore\")\n",
-    "    result = tuner.tune(\n",
+    "    result, best_model = tuner.tune(\n",
```

Comment:
If we include best model in the namedtuple, we dont need to change this as well.